docs/content/modules/tensor_mechanics/systems/Kernels/StressDivergence1DRSpherical.md

```diff
@@ -1,4 +1,4 @@
-<!-- MOOSE System Documentation Stub: Remove this when content is added. -->
-#StressDivergence1DRSpherical System
-!parameters /Kernels/StressDivergence1DRSpherical
+#Stress Divergence 1D RSpherical Action
```

Comment:
I would leave the long name for now, we will be improving this sidebar in the coming weeks. I am hoping we can handle the long names. So, it is better to leave it for now so that it matches the object name.